CHT/HeatFlux.C

```diff
@@ -31,25 +34,60 @@ void preciceAdapter::CHT::HeatFlux::write(double * buffer)
     {
         int patchID = patchIDs_.at(j);
 
-        // Extract the effective conductivity on the patch
-        extractKappaEff(patchID);
+        //TODO: Ask for the locationsType, currently Triangles assumed
```

Comment:
Does this still hold?

Reply:
And, most importantly, is it expected to break any functionality?

Reply:
No, this is deprecated. It doesn't break anything.